### DISCUSSION TITLE
大谷翔平加入道奇隊

### DISCUSSION THREAD

**Discussion by uwje** [1101-GS4719-C/Python-Community-GS4719-C#1768] [General]
日籍大聯盟球星大谷翔平，今日官宣加入道奇，以十年七億美金的薪資加入，但攤開來看，這份合約其實暗藏玄機，前十年其實年薪只有200位美金，使球隊能夠有空間去補強，增加奪冠可能性。
我認為這樣是好的，但我擔心會有更多人會效仿，導致某個球隊過強，導致其他球隊失去競爭力。而且等他退休後，他無法為球隊提供戰力，但卻仍佔球隊薪資空間，可能導致球隊薪資空間不夠。
但他這樣也釋出一個訊息，他真的想拿冠軍。ㄦ
參考連結：https://www.cna.com.tw/news/aspt/202312120077.aspx

**Comment by mrweixd** [comment]
大谷翔平以低年薪加入道奇，顯示他為球隊的整體利益願意做出妥協，以增加奪冠可能性。然而，這樣的合約模式引發擔憂，可能造成其他球員效仿，使某些球隊過於強大，影響整體聯盟的競爭公平。此外，他的退休後可能對球隊薪資空間產生影響，需要謹慎平衡球隊發展和個別球員合約的長遠影響。

**Comment by uwje** [comment]
這樣雖然不違反現在的規則，但不可否認的是這樣的合約會破壞聯盟的平衡，導致某些球隊過於強勢，希望聯盟能認識到相關問題並提出相關規定去限制。
